src/Streamly.hs

```diff
@@ -156,12 +156,12 @@ module Streamly {-# DEPRECATED "Please use \"Streamly.Prelude\" instead." #-}
     -- $adapters
     , IsStream ()
 
-    , serially
-    , wSerially
-    , asyncly
-    , aheadly
-    , wAsyncly
-    , parallely
+    , fromSerial
+    , fromWserial
+    , fromAsync
+    , fromAhead
+    , fromWasync
+    , fromParallel
```

Comment:
Since these are already released combinators, we should just deprecate these in this module rather than changing them. We do not need to have the new names in this module.

On the other hand in the "Streamly.Prelude" module these combinators have been introduced in 0.8.0 only so there we can just change them, and not deprecate them.